src/main/java/duke/models/locker/Locker.java

```diff
@@ -218,4 +224,79 @@ public boolean equals(Object other) {
     public int hashCode() {
         return Objects.hash(serialNumber, address, zone, tag);
     }
+
+    /**
+     * This function is used to compare the locker info with a locker that was searched.
+     * This is used in conjunction with Java in-streams.
+     * @param findLocker has all or some the attributes the locker that was searched for.
+     * @param findStudent has all or some the details of the student that was searched for.
+     * @return refers to a boolean value to check if the comparison was true or false.
+     */
+
+    public boolean compare(FindCommand.FindLocker findLocker, FindCommand.FindStudent findStudent) {
```

Comment:
Just curious .Is it possible to move this entire function inside findCommand? because since both function parameters are just like directly related to find locker and in a way got nothing to do with the model "locker" itself?